corehq/apps/enterprise/templates/enterprise/project_dashboard.html

```diff
@@ -1,25 +1,23 @@
 {% extends "hqwebapp/bootstrap5/base_section.html" %}
 {% load hq_shared_tags %}
 {% load i18n %}
-
-{% block page_title %}{{ current_page.title }}{% endblock %}
-
 {% js_entry 'enterprise/js/project_dashboard' %}
 
 {% block page_content %}
   {% registerurl "enterprise_dashboard_email" domain "---" %}
   {% registerurl "enterprise_dashboard_total" domain "---" %}
-
   {% initial_page_data 'max_date_range_days' max_date_range_days %}
   {% initial_page_data 'metric_type' metric_type %}
 
-  <div class="row mt-4">
-    {% for report in reports %}
-      <div class="col-md-6 col-lg-6 col-xl-3 mb-3">
-        {% include 'enterprise/partials/project_tile.html' with report=report uses_date_range=uses_date_range %}
-      </div>
-    {% endfor %}
-  </div>
-
+  {% for group in groups %}
+    <div class="row {% if not forloop.first %}mt-4{% endif %}">
```

Comment:
Rather than conditionally include a top margin, would it be easier to have each row instead have this as a bottom margin?

Reply:
This is also reverted since we're bringing the page title back, I need some top margin even for the first row.